### PR TITLE
Travis Fix for E2E tests

### PR DESCRIPTION
**Pull request by codevulture**
#258

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -9,7 +9,7 @@ deps = -r{toxinidir}/test-requirements.txt
        -r{toxinidir}/requirements.txt
 commands =
    python -V
-   nosetests []
+   nosetests -e e2e -v
```
Comment (mbohlool) [inline tox.ini]: Is this fixing #258? Can you explain how?

**Comment by codevulture** [comment]
@mbohlool : yes, Please find the logs of checks after this patch in TOXENV=27, others: 
[Travis Checks after this patch](https://travis-ci.org/kubernetes-incubator/client-python/builds/244202874?utm_source=github_status&utm_medium=notification
)
and before this travis run of pull request say: 
[Travis Pull request checks before this patch](https://travis-ci.org/kubernetes-incubator/client-python/builds/240465104?utm_source=github_status&utm_medium=notification)
in which error/warning logs could be clearly seen in TOXENV=27,others.

**Comment by mbohlool** [comment]
Yes, I looked at those logs. I am more curious to know what does this change do.

**Comment by mbohlool** [comment]
I see "-e" excludes e2e test. does that mean any test starts with e2e? should we also add `[]` at the end?

**Comment by codevulture** [comment]
@mbohlool : yes, -e works as a regex on tests list generated which excludes e2e tests. Also there is no need to add [] in end. If needed i can remove it from places where it is used in different patch.

**Comment by mbohlool** [comment]
Looks good. Branch is out of date. Can you rebase?

**Comment by codevulture** [comment]
@mbohlool : great! Updated.

**Comment by mbohlool** [comment]
Thanks

**Pull request merged**